Fix "no .eh_frame_hdr table will be created" error.

http://gcc.gnu.org/bugzilla/show_bug.cgi?id=47829
http://gitweb.dragonflybsd.org/dragonfly.git/commitdiff/4a0e0cd7377c45dc21d5c71bcab931b5e5727e74

diff -Nur lib.orig/csu/i386-elf/Makefile lib/csu/i386-elf/Makefile
--- lib.orig/csu/i386-elf/Makefile	2013-03-06 01:07:57.000000000 +0900
+++ lib/csu/i386-elf/Makefile	2013-03-06 01:10:09.000000000 +0900
@@ -10,6 +10,8 @@
 FILESDIR=	${LIBDIR}
 CFLAGS+=	-I${.CURDIR}/../common \
 		-I${.CURDIR}/../../libc/include
+CFLAGS+=	-fno-omit-frame-pointer
+CFLAGS+=	-fno-asynchronous-unwind-tables
 CLEANFILES=	${FILES} crt1_c.o crt1_s.o gcrt1_c.o Scrt1_c.o
 CLEANFILES+=	crt1_c.s gcrt1_c.s Scrt1_c.s
 
